src/FirstSteps/RPG/Heroes/Boss.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+using System.Dynamic;
+
+namespace FirstSteps.RPG.Heroes
+{
+    public class Boss : IAttacker
+    {
+        public string Name;
```

Comment:
`public string Name;` is a public field that everyone can change. You should use property with getter to make it read-only. It is applicable to all fields below.